src/cmd/builtin/version.c

```diff
@@ -57,6 +62,12 @@ static int cmd_version (optparse_t *p, int ac, char *av[])
 #endif
 #if HAVE_CALIPER
     printf ("+caliper");
+#endif
+#if HAVE_LIBPMIX
+    printf ("+pmix==%ld.%ld.%ld",
```

Comment:
Do we want to be explicit that this is the version of the client-side that we are using?  I'm afraid users might think this is the version of pmix that Flux provides as a server.

Maybe `+pmix` -> `+pmix-bootstrap`?

Reply:
That works for me if it helps you!